lockfile.json

```diff
@@ -189,7 +189,7 @@
     "revision": "fcc5f6f4d194dede4e676834ff28a506e39e17b4"
   },
   "tlaplus": {
-    "revision": "63ab99903cce883e8db392888c526fe2ffd00b1a"
+    "revision": "6130577a46342c1f9dfb6f27f543313fce7bb091"
```

Comment:
Looks like queries need to be updated for this parser

Reply:
/cc @ahelwer

Reply:
Correct! I issued a new release; the query files are kept in the `nvim` dir of the same repo and are tested by CI. So this process automatically tries to update the grammar versions to the latest main/master commit?

Reply:
Either against this PR or just against master (this PR will be automatically updated)